pingcap/tidb
# *: add affected rows metrics

Signed-off-by: Jack Yu <jackysp@gmail.com>



### What problem does this PR solve?


Issue Number: close #36435

Problem Summary: 

### What is changed and how it works?

Add metrics of affected rows.

### Check List

Tests 


1. start a cluster
2. run sysbench prepare data
3. check Grafana -> TiDB -> Server -> Affected Rows by Type
<img width="791" alt="image" src="https://user-images.githubusercontent.com/4352397/180631853-a9aab162-67b5-407c-abfa-9aa58ff0a54e.png">
4. compare with QPS panel, this new panel could show the real number of written rows.
<img width="798" alt="image" src="https://user-images.githubusercontent.com/4352397/180631879-b40d2a9e-2256-416e-b525-a429fbc89b66.png">



Side effects


Documentation


### Release note



Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```
